kotlin-checks-test-sources/src/main/kotlin/checks/UnusedDeferredResultCheckSample.kt

```diff
@@ -9,28 +9,28 @@ import kotlinx.coroutines.delay
 
 suspend fun unused() {
     
-    coroutineScope { // Noncompliant {{This function returns "Deferred", but its result is never used}} 
+    coroutineScope { // Noncompliant {{This function returns "Deferred", but its result is never used.}}
```

Comment:
Can we remove all locations checking the message except for the first one? The message here is always the same, so I see no value in asserting it every time - it just creates more work when changing it.